dev/preview/previewctl/cmd/get_name.go

```diff
@@ -12,12 +12,19 @@ import (
 	"github.com/gitpod-io/gitpod/previewctl/pkg/preview"
 )
 
-func getNameCmd() *cobra.Command {
+func newGetNameCmd(branch string) *cobra.Command {
```

Comment:
`newGetNameCmd` looks a bit like a temporary name that allows the old function and a new one to co-exist. Since this PR only contains one function - we don't need the `new` prefix, do we?

Reply:
I'll rename the rest in another PR so they all are named `New...`, as it is a better name considering their function

Reply:
👍